### DISCUSSION TITLE
Purchased Cyberduck APP not completing download

### DISCUSSION THREAD

**Discussion by LGB3839** [iterate-ch/cyberduck#15926] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

**Comment by LGB3839** [comment]
After purchasing the Cyberduck app in the Apple store, the download stopped at 72%.  My credit card was charged and I need help with resolving this issue. I am on a Mac Mini using Sonoma OS.

Was finally able to complete download after connecting via an ethernet cord and opened Cyberduck but am now getting this message: 
> Login sftp2.infinedi.net – SFTP with username and password. Please contact your web hosting service provider for assistance.

**Reply by tatze96** [reply]
Please add a screenshot of the mentioned error message. Make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
